<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CargarFacultad</class>
 <widget class="QWidget" name="CargarFacultad">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cargar Producto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/images/icons/cil-pencil.png</normaloff>:/icons/images/icons/cil-pencil.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color: #333;
	font: 9pt &quot;Segoe UI&quot;;
}
/* ----------------- LineEdit  ----------------- */
QLineEdit {
	border-radius: 5px;
	border: 2px solid #1F54A3;
	padding-left: 10px;
	selection-color: white;
	selection-background-color: #2A3C57;
    color: black;
}
QLineEdit:hover {
	border: 2px solid rgb(98, 114, 164);
}
QLineEdit:focus {
	border: 2px solid #2A3C57;
}



/* ----------------- Button ----------------- */
QPushButton {
	border: 2px solid #1F54A3;
	border-radius: 5px;	
	background-color: #1F54A3;
	color:white;
}
 QPushButton:hover {
	background-color: #286FD6;
	border: 2px solid #286FD6;
}
 QPushButton:pressed {	
	background-color: #2A3C57;
	border: 2px solid #2A3C57;
}

/* ----------------- CheckBox ----------------- */

QCheckBox::indicator {
	border : 1px solid #6272a4;
	width: 15px;
	height: 15px;
	border-radius: 15px;
    background: #6272a4;
}
QCheckBox::indicator:hover {
	background-color: #286FD6;
	border : 1px solid #286FD6;
}
QCheckBox::indicator:checked {
	border : 1px solid #1F54A3;
    background: 3px solid #1F54A3;	
	background-image: url(:/icons/images/icons/cil-check-alt.png);
}


/* ----------------- QDateEdit ----------------- */
QDateTimeEdit{
	background-color: white;
	border-radius: 5px;
	border: 2px solid #1F54A3;
    color: black;
}
QDateTimeEdit:hover{
	border: 2px solid #7284b9;
}

QDateTimeEdit:focus {
	border: 2px solid #2A3C57;
}


QDateTimeEdit::up-button {
    border-image: url(:/icons/images/icons/cil-arrow-top.png); 
}

QDateTimeEdit::down-button {
    border-image: url(:/icons/images/icons/cil-arrow-bottom.png);
}














</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>16</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 16pt &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Agregar Carrera</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Nombre Carrera</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txt_nombreCarrera">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>295</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Facultad</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_codCarrera">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>295</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Codigo Carrera</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cboFacultad"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_AnadirCarrera">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>AÃ±adir</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_nombreCarrera</tabstop>
  <tabstop>btn_AnadirCarrera</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
